Fixed bug to allow search to return more than 10 items

shengdi opened the pull request:
I have been using gplaycli to search and download large numbers of apps from the app store. But recently the search only returns a maximum of 10 items. This pull request makes some quick and shoddy changes to request for the next pages and combine the items together to return more matching applications based on "-n", this breaks API compatibility with googleplay_api though, but that code is not being maintained anymore.

matlink commented:
Well, how did you implemented it? `-n` does not return the expected number of results.
```
$ gplaycli -n 20 -s firefox | wc -l
48
```

shengdi commented:
Ah sorry, so basically I added a loop to request for the next page `message.payload.searchResponse.doc[0].containerMetadata.nextPageUrl`, but it seems like the number of applications returned actually varies from the first page to the next, added counting to make sure -n returns the right number.

Should be working properly now.

matlink commented on ext_libs/googleplay_api/googleplay.py:
```diff
@@ -194,13 +194,27 @@ def executeRequestApi2(self, path, datapost=None, post_content_type="application
+        allmessages = message
+
+        while remaining > 0:
+            print(remaining)
```
don't need this print

matlink commented on ext_libs/googleplay_api/googleplay.py:
```diff
@@ -194,13 +194,27 @@ def executeRequestApi2(self, path, datapost=None, post_content_type="application
+          
+          
+
+        
```
remove these blank lines

matlink commented:
Hum, I'll revert it, because it sometimes throws errors: 
```
$ gplaycli -s "com.yogavpn"
Traceback (most recent call last):
[...]
    remaining -= len(messagenext.payload.searchResponse.doc[0].child)
IndexError: list index out of range

$ gplaycli -s "com.yogavpn" -n 15
Traceback (most recent call last):
[...]
    remaining -= len(messagenext.payload.searchResponse.doc[0].child)
IndexError: list index out of range
```
